plugin-niassa/src/main/java/ca/on/oicr/gsi/shesmu/niassa/CustomInputLimsKeyProvider.java

```diff
@@ -48,4 +48,15 @@ public void store(WorkflowAction action, Object value) {
       }
     };
   }
+
+  public CustomActionParameter<MigrationAction> parameterMigration() {
+    return new CustomActionParameter<MigrationAction>("wdl_outputs_migration", true, type) {
```

Comment:
I think this should stay `wdl_outputs`. It' used for generating the INI, so it might not matter at all.